src/main/builders/formBuilder.ts

```diff
@@ -14,10 +15,15 @@ export class FormBuilder {
     validationErrors: { [key: string]: string } = {}
   ): void | Response {
     let template = '';
+    const listOfValuesLength = JSON.parse(config.get('radio.listOfValuesLength'));
     //check the types here
-    if (parent?._listOfValuesLength && parent?._listOfValuesLength > 0 && parent?._listOfValuesLength < 10) {
+    if (
+      parent?._listOfValuesLength &&
```

Comment:
This will be truthy only when `parent` is defined and `_listOfValuesLength` is non-zero, meaning that if this resolves to true, next line should also be always true. Though maybe it is good to be rather explicit here.